tests/test-basic-c.c

```diff
@@ -236,20 +236,140 @@ test_object_writes (gconstpointer data)
   }
 }
 
+static GVariant*
+xattr_cb (OstreeRepo  *repo,
+          const char  *path,
+          GFileInfo   *file_info,
+          gpointer     user_data)
+{
+  GVariant *xattr = user_data;
+  if (g_str_equal (path, "/baz/cow"))
+    return g_variant_ref (xattr);
+  return NULL;
+}
+
+/* check that using a devino cache doesn't cause us to ignore xattr callbacks */
+static void
+test_devino_cache_xattrs (void)
+{
+  g_autoptr(GError) error = NULL;
+  gboolean ret = FALSE;
+
+  g_autoptr(GFile) repo_path = g_file_new_for_path ("repo");
+
+  /* re-initialize as bare */
+  ret = ot_test_run_libtest ("setup_test_repository bare", &error);
+  g_assert_no_error (error);
+  g_assert (ret);
+
+  gboolean on_overlay;
+  ret = ot_check_for_overlay (&on_overlay, &error);
+  g_assert_no_error (error);
+  g_assert (ret);
+
+  if (on_overlay)
+    {
+      g_test_skip ("overlayfs detected");
```

Comment:
This is fine for now, though a more accurate fix would be porting the "can relabel" logic.  Though OTOH I agree it is perhaps easier for dev testing to set the directory.  Maybe we want TMPDIR_PREFIX though?